### PR TITLE
Release v0.1.2

### PR DESCRIPTION
**Pull request by taylorbot**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.2] - 2023-07-27
+
```
Comment (pipo02mix) [inline CHANGELOG.md:11]:
```suggestion
## [0.1.2] - 2023-07-27

### Added

- A page for incident management FAQ.

```

**Review by MarcelMue — APPROVED** [review]

**Review by lyind — APPROVED** [review]
LGTM

**Comment by pipo02mix** [comment]
there is a link to a private repo, can I skip the validation for this link?

**Comment by lyind** [comment]
> there is a link to a private repo, can I skip the validation for this link?

In the "Check Markdown links" check! Didn't see it. 🤦

I'm not sure the private repo link should be public if the repository itself also isn't.

@MarcelMue Would it make sense for the Markdown Link Checker to support Authentication, is it possible or already configured? We already have the performance improvement issue here and could potentially implement passing through authentication cookies for some URL patterns: https://github.com/giantswarm/giantswarm/issues/27480

Previous issues mentioning the need for `markdown-link-check` authentication support/workarounds:
- https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/58
- https://github.com/tcort/markdown-link-check/issues/18

**Comment by lyind** [comment]
Maybe the `ignorePatterns` which can be passed via `config-file` (see [https://github.com/gaurav-nelson/github-action-markdown-link-check#custom-variables](https://github.com/gaurav-nelson/github-action-markdown-link-check#custom-variables)) could help?

**Pull request merged**